src/Components/AddRoom/AddRoom.tsx

```diff
@@ -0,0 +1,221 @@
+                  <Grid item xs={12} sx={{ color: "#17202A" }} >
+                    <TextField
+                      InputLabelProps={{
+                        style: { color: 'black' },
+                      }}
+                      sx={{ bgcolor: "#F7F7F7", mb: 5, }}
+                      variant='filled'
+                      label="Room Number"
+                      fullWidth
+                      {...register("roomNumber",{required:true})}
+                    />
+                    {errors.roomNumber&&errors.roomNumber.type==="required"&& <Typography sx={{color:"red"}}>Room Number is required</Typography>}
+
+                  </Grid>
+                </Grid>
+              </Box>
+
+              <Box sx={{ width: '100%' }}>
+                <Grid container rowSpacing={1} columnSpacing={{ xs: 2, sm: 2, md: 2 }}>
+                  <Grid item xs={6} >
+                    <TextField
+                      InputLabelProps={{
+                        style: { color: 'black' },
+                      }}
+
+                      variant='filled'
+                      sx={{
+                        bgcolor: "#F7F7F7", display: "flex",
+                        justifyContent: "flex-start"
+                      }}
+                      label="Price"
+                      {...register("price",{required:true})} />
+                    {errors.price&&errors.price.type==="required"&& <Typography sx={{color:"red"}}>price is required</Typography>}
+
+                  </Grid>
+                  <Grid item xs={6}>
+                    <TextField
+                      InputLabelProps={{
+                        style: { color: 'black' },
+                      }}
+                      sx={{
+                        bgcolor: "#F7F7F7", mb: 5, display: "flex",
+                        justifyContent: "flex-end",
+                      }}
+                      variant='filled'
+                      label="Capacity"
+                      {...register("capacity",{required:true})}
+                    />
+                    {errors.capacity&&errors.capacity.type==="required"&& <Typography sx={{color:"red"}}>capacity is required</Typography>}
+
+                  </Grid>
+                  <Grid item xs={6}>
+                    <TextField
+                      InputLabelProps={{
+                        style: { color: 'black' },
+                      }}
+                      sx={{
+                        bgcolor: "#F7F7F7", mb: 5, display: "flex",
+                        justifyContent: "flex-start",
+                      }}
+                      variant='filled'
+                      label="Discount"
+                      {...register("discount",{required:true})}
+                    />
+                    {errors.discount&&errors.discount.type==="required"&& <Typography sx={{color:"red"}}>discount is required</Typography>}
+
+
+                  </Grid>
+                  <Grid item xs={6}>
+
+                    <Select
```

Comment:
We need to add a label for this Select or make the first option disabled with label "Select Facilities" instead of "Select"